services/vod-to-live/service/src/domains/validation/setup-pre-publishing-validation-webhook.ts

```diff
@@ -0,0 +1,87 @@
+import {
+  Dict,
+  generateWebhookResponse,
+  handleWebhookErrorMiddleware,
+  Logger,
+  verifyWebhookRequestMiddleware,
+  WebhookRequestMessage,
+  WebhookResponse,
+} from '@axinom/mosaic-service-common';
+import { Express, Request, Response } from 'express';
+import { Config, ValidationErrors } from '../../common';
+import { AzureStorage } from '../azure';
+import { ChannelPublishedValidationWebhookHandler } from './channel-published-validation-handler';
+import { PlaylistPublishedValidationWebhookHandler } from './playlist-published-validation-handler';
+import {
+  PrePublishingPayload,
+  ValidationWebhookHandler,
+} from './validation-webhook-model';
+
+const logger = new Logger({
+  context: 'Pre-Publishing-Validation-Webhook-Handler',
+});
+
+export const setupPrePublishingValidationWebhook = (
+  app: Express,
+  config: Config,
+  storage: AzureStorage,
+): void => {
+  const handlers: ValidationWebhookHandler<PrePublishingPayload>[] = [
+    new ChannelPublishedValidationWebhookHandler(config),
+    new PlaylistPublishedValidationWebhookHandler(config, storage),
+  ];
+  app.post(
+    '/pre-publishing',
+    verifyWebhookRequestMiddleware({
+      webhookSecret: config.prePublishingWebhookSecret,
+    }),
+    handleWebhookErrorMiddleware,
+    async (
+      req: Request<
+        Dict<string>,
+        Dict<string>,
+        WebhookRequestMessage<PrePublishingPayload>
+      >,
+      res: Response,
+    ) => {
+      const requestBody = req.body;
+      let validationResponse: WebhookResponse<PrePublishingPayload> | null =
```

Comment:
minor: can be set to just `let validationResponse: WebhookResponse<PrePublishingPayload>;`